csrankings-m.csv

```diff
@@ -1986,14 +1987,15 @@ Miltos Petridis,Middlesex University,http://www.cs.mdx.ac.uk/people/miltos-petri
 Mimi Zhang,Trinity College Dublin,https://www.tcd.ie/research/profiles/?profile=zhangm3,6NC-XagAAAAJ
 Mimmo Parente,University of Salerno,http://www.di.unisa.it/~parente,45LUzAwAAAAJ
 Mimi Xie,University of Texas at San Antonio,https://cs.utsa.edu/people/faculty/xie-mimi-936,Za4_qhkAAAAJ
+Min C. Shin,UNC - Charlotte,https://sites.google.com/a/uncc.edu/mcshin/,BVPAmrwAAAAJ
 Min Cao,Soochow University,http://web.suda.edu.cn/cm,nhMWtZsAAAAJ
 Min Chi,North Carolina State University,https://www.csc.ncsu.edu/people/mchi,gJmbChYAAAAJ
 Min Chih Lin,University of Buenos Aires,http://www.ic.fcen.uba.ar/~mlin/homepage.html,NOSCHOLARPAGE
 Min H. Kim 0001,KAIST,http://vclab.kaist.ac.kr/minhkim,pMT5lrwAAAAJ
 Min Hun Lee,Singapore Management University,https://scis.smu.edu.sg/faculty/profile/6256/lee-min,quDiEBkAAAA
 Min Long,Boise State University,http://flash.uchicago.edu/~long,Ex_raoQAAAAJ
 Min Lu 0002,Shenzhen University,http://deardeer.github.io,xFAU798AAAAJ
-Min Shin,UNC - Charlotte,https://viscenter.uncc.edu/min-shin,T50sdQ8AAAAJ
+Min Shin,UNC - Charlotte,https://sites.google.com/a/uncc.edu/mcshin/,BVPAmrwAAAAJ

```

Comment:
Not the correct DBLP entry, it appears. See https://dblp.org/pid/67/341.html.

Reply:
Min Shin had 2 of 3 publications associated with the Min C. Shin at UNC - Charlotte. I requested dblp to correct this. I will delete.